server/src/main/java/com/cloud/server/StatsCollector.java

```diff
@@ -1364,6 +1364,12 @@ public boolean imageStoreHasEnoughCapacity(DataStore imageStore) {
         return false;
     }
 
+    public Long imageStoreCurrentCapacity(DataStore imageStore) {
```

Comment:
The name is misleading, moreover `imageStoreHasEnoughCapacity` can be used instead.